yarn-project/prover-node/src/prover-node-publisher.ts

```diff
@@ -176,8 +180,18 @@ export class ProverNodePublisher {
       );
     }
 
+    // Check the batched blob inputs from the root rollup against the batched blob computed in ts
+    // TODO(MW): There is probably a way to add the opening proof Q to publicInputs, but it's not needed in the circuit => may be confusing
```

Comment:
Yeah I think it'd be confusing if the opening proof Q appeared in a circuit public inputs struct. What was the reason for you to contemplate to add it?

Reply:
Because I needed to add `batchedBlobInputs: BatchedBlob;` as an extra input to each fn in this file, and as an extra output from the orchestrator, which (I think) bloats the code a bit considering I'm doing all that just for inlcuding a single point (we have everything else from the circuits IIRC). But on balance it does seem better to keep it this way, thanks!